src/vos/tests/vts_io.c

```diff
@@ -221,8 +256,11 @@ io_recx_iterate(struct io_test_args *arg, vos_iter_param_t *param,
 
 	rc = vos_iter_prepare(itype, param, &ih);
 	if (rc != 0) {
-		print_error("Failed to create recx iterator: %d\n", rc);
-		goto out;
+		if (rc == -DER_NONEXIST)
```

Comment:
Then the subsequent -DER_NONEXIST check for vos_iter_probe() return value can be removed.

Reply:
I don't think so.   The first one happens if no tree exists.   The 2nd one would happen if the tree is empty.